src/components/NewMessage/NewMessage.vue

```diff
@@ -515,8 +515,9 @@ export default {
 			}
 
 			if (this.upload) {
-				// Clear input content from store
+				// Clear input content from store and remove Quote component
 				this.chatExtrasStore.setChatInput({ token: this.token, text: '' })
+				this.chatExtrasStore.removeParentIdToReply(this.token)
```

Comment:
Add a check in `removeParentIdToReply()` for the existence of the token item in the store.

Reply:
Vue has an inner guard for it, so nothing will happen if you call `Vue.delete(this.parentToReply, token)` and `this.parentToReply[token]` is undefined